content/en/docs/tasks/debug/debug-application/debug-running-pod.md

```diff
@@ -610,8 +610,7 @@ kubectl delete pod myapp myapp-debug
 
 ## Debugging via a shell on the node {#node-shell-session}
 
-If none of these approaches work, you can find the Node on which the Pod is
-running and create a privileged Pod running in the host namespaces. To create
+If none of the previous approaches work, you can find the Node on which the Pod is running and create a debug Pod on that node. This Pod runs as `root` and has access to the Node's filesystem through `/host`. However, it is not a fully privileged pod. Limitations include not being able to `chroot /host` and reading some process information may fail. To create
```

Comment:
```suggestion
If none of the previous approaches work, you can find the Node on which the Pod is running and
create a debug Pod on that node. This Pod runs as `root` and has access to the Node's filesystem
through `/host`. However, it is not a fully privileged pod. Limitations include not being able to
`chroot /host` and reading some process information may fail. To create
```

Reply:
@tengqm Would you like to remove the words which are highlighted in red?

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question

Reply:
@RA489 thanks! I got it

Reply:
@verb 

> Please link to https://kubernetes.io/docs/concepts/workloads/pods/#privileged-mode-for-containers to define what privileged means to Kubernetes

Is the debug node privileged in that sense though, since the pod [isn't created with the privileged flag](https://github.com/kubernetes/kubectl/blob/f125772/pkg/cmd/debug/debug.go#L567)?

Maybe the easier solution here is to remove the word `privileged` from this discussion? Although it is privileged in a sense since it has access to the node filesystem through `/host`. It's a strange sort of pseudo-privileged.